Directory.Packages.props

```diff
@@ -197,6 +197,7 @@
     <!-- Pinned versions for Component Governance - Remove when root dependencies are updated -->
     <PackageVersion Include="Azure.Core" Version="1.44.0" />
     <PackageVersion Include="Azure.Identity" Version="1.12.1" />
+    <PackageVersion Include="System.Runtime.Caching" Version="8.0.1" />
```

Comment:
```suggestion
```

This is not necessary. See https://github.com/dotnet/aspire/issues/6226#issuecomment-2412218950